src/assets/layout.scss

```diff
@@ -2,19 +2,28 @@
  * layout.scss
  */
 html {
-  /* We can avoid setting height:100vh on the grid container if we use height:100% 
-     on the root element and then all descendants up until the grid container. 
-     Using 100vh can be a simpler option, but the vh unit can be problematic on 
-     iOS and Android: https://nicolas-hoizey.com/articles/2015/02/18/viewport-height-is-taller-than-the-visible-part-of-the-document-in-some-mobile-browsers/ 
+  /* We can avoid setting height:100vh on the grid container if we use height:100%
+     on the root element and then all descendants up until the grid container.
+     Using 100vh can be a simpler option, but the vh unit can be problematic on
+     iOS and Android: https://nicolas-hoizey.com/articles/2015/02/18/viewport-height-is-taller-than-the-visible-part-of-the-document-in-some-mobile-browsers/
    */
   height: 100%;
   /* With our styles we should not have any cause for a scrollbar on the page itself,
      so this is not strictly necessary, but to be safe we can prevent scrollbars on
-     the full page. 
+     the full page.
    */
   overflow: hidden;
 }
 
 body {
   height: 100%;
 }
+
+.full-screen {
+  position: absolute !important;
```

Comment:
;'(

Reply:
yeh... but this time it's `reelly !important LUL`